parameters/experiments/m6/pursuit-with-noise.params

```diff
@@ -2,7 +2,7 @@ _includes:
    - "../../root.params"
    - "pursuit-nouns.m6.params"
 
-output_directory: '%adam_experiment_root%/m6/deniz'
+output_directory: '%adam_experiment_root%/'
```

Comment:
We should never output directly to the experiment root because the users might have other experiments besides this. Put this in an `m6` sub-directory which will be the root of our archive.